tools/pubsys/src/aws/ssm/ssm.rs

```diff
@@ -239,7 +239,12 @@ pub(crate) async fn set_parameters(
                 // RusotoError::Unknown(BufferedHttpResponse {status: 400, body: "{\"__type\":\"ThrottlingException\",\"message\":\"Rate exceeded\"}", headers: ...})
                 // Even if we were to do a structural match, we would still have to string match
                 // the body of the error.  Simpler to match the string form.
-                if e.to_string().contains("ThrottlingException") {
```

Comment:
I can't comment on lines 239-241 but the comment mentions Rusoto and is probably outdated.